modules/islandora_iiif/src/Plugin/views/style/IIIFManifest.php

```diff
@@ -494,17 +494,20 @@ protected function defineOptions() {
    */
   protected function addSearchEndpoint(array &$json, array $url_components) {
     $url_base = $this->getRequest()->getSchemeAndHttpHost();
-    $hocr_search_path = $this->options['search_endpoint'];
-    $hocr_search_url = $url_base . '/' . ltrim($hocr_search_path, '/');
+    $hocr_search_path = $this->options['search_endpoint'] ?? null;
```

Comment:
Looks like this introduced a little coding standards thing: https://github.com/Islandora/islandora/pull/1040